rest-twitter/service/TweetRetriever_Test.py

```diff
@@ -3,23 +3,17 @@
 # Make sure to run with e.g. "env TWITTER_API_KEY=xxx TWITTER_API_KEY_SECRET=xxx python TweetRetriever_Test.py"
 
 from TweetRetriever import TweetRetriever
-from vaderSent import SentimentAnalyzer
-import time
-from config.location_service_config import CACHE
-from service.Location_Service import *
-import os
```

Comment:
Removed integration with sentiment analyzer in `TweetRetriever_Test.py` since imports no longer worked after we restructured the code, and the test of TweetRetriever should be as specific to TweetRetriever as possible.